### ISSUE TITLE
Increasing the incentives of consolidation.

### ISSUE THREAD

**Issue by ydennisy** [WICG/first-party-sets#13]
Thank you for publishing this proposal.

Whilst from a technical perspective this type of proposal makes perfect sense, domains owned by a single organisation should have a simple and clean way to share information about users across them. 

However from a market perspective I feel this could have negative effects on the web:
- Large players such as `google.com` & `youtube.com` have a huge advantage when sharing data across two such domains, the level of data and granularity here will be much more than needed to compound the "creepy ads problem".
- This could force/incentivise large publishers to consolidate in an attempt compete with domains such as `google.com` & `facebook.com`, meaning less diversity especially in news publishers. Whilst also exaggerating the level of granular profiling companies are able to do.
- Niche publishers are at a huge disadvantage in terms of advertising budgets in the model, as their data is not enough to really build interesting profiles over and above contextual information.

So while doing something such as this will limit the overall number of sites able to track users across multiple domains, it will have:
- No real positive outcome for users on privacy, as having access to data from a few `super` domains is enough.
- Negative effect on the overall diversity of the "open" web.

**Comment by joshuakoran** [comment]
Great question. I believe your question highlights the perspective that limiting data collection/processing to a first-party has a positive effect on privacy.  To better focus the discussion on first-party sets, perhaps we should list the risks/harms posed by third-party data collection/processing from which we want to protect end users and see which of these could also be done by sets of "bad" first-parties?

**Comment by ydennisy** [comment]
@joshuakoran Yes, but I am not really considering "bad" first parties, I am merely stating that if we have a very global goal of increased user privacy on the web. Then we must consider, the difference between:
- a huge global org, tying together domains which would capture granular data from >90% of internet usage
- medium sized orgs, who will be incentivised to join forces to create a bigger "first party set" to rival the huge orgs.
- small sized orgs, who will be hugely disadvantaged, without much ability to compete.

**Comment by jeffreytgilbert** [comment]
Correct me if I'm wrong here, but the intent of this is to align domains with top level owners. I don't believe there is an established goal of stopping businesses from merging to create bigger businesses. The challenge is solving shared authentication over different top eTLD+1 domains. 

While I share your concern about cutting the legs out from under small publishers, they have never had an ability to measure an audience of identified users across multiple sites without using a large DMP. Siloing identity as a result of two factors (who and what/where) and limiting the scope of that information is the actual cause of disadvantaging small publishers. Enormous marketplaces and publishers such as those mentioned in your opening post stand to gain the most if they already have critical mass and stickiness. Stand alone DMPs will be most heavily impacted by tracking prevention as their data will lose value as tracking across multiple domains is no longer a free for all. 

Today, even large publishers struggle to compete with big data. A large buying, selling, or data management platform are quite capable freely feeding on information without limit. Low quality publishers gain the most from freely shared identity. Those sites can monetize junk content without ever having a user indicate they have a trust relationship with publisher. 

My point is, the goal here should not be to severely impact big business or dissuade consolidation effects. The important thing here is restoring the relationship a user has with a trusted brand, regardless of their size. The only way to do that is to restrict access to identity and allow properly scoped sharing of that information. Regulations like GDPR, DSR, CCPA, etc are probably the only way to limit data flow between businesses to appropriate use only. Today, the main issue is limiting the exposure to freely available trackable information. 

What I should be able to do as an end user is log into a service and see where my information can be shared at that point in time. That is an explicit contract between me, the user, and the domain(s) or businesses in question. Surfacing and standardizing the display and clarity of intent regarding this information should be priority 1. 

Example:

Step 1: Initiate call to establish users trust between themself and the publishers/domains

DomainAccessProxy.request( {
 session: 29348857483423948,
 domains: ['example.com', '.example.co.uk', '.example.jp'],
 proposed-duration: 86500,
 interfaces: ['cookie', 'microphone', 'microphone', ...],
 access-type: 'read-write'
 } ); // -> promise

Step 2: Establish verifiable boundary of trust between publishers using FPS

Fetch -> https://example.com/fps-manifest.json

[
      {"entity": { "domain": "example.com", "certificate-signature": "23849jnskjndj...", "duration": 86500 },
      {"entity": { "domain": "example.co.uk", "certificate-signature": "23849jnskjndj...", "duration": 86500 },
      {"entity": { "domain": "example.jp", "certificate-signature": "23849jnskjndj...", "duration": 86500 }
]

Step 3: Present the permissions modal to the user via native browser UI

Hello Jeffrey,
Please indicate the access you'd like to grant for the following requests:
 [x]  example.com (this domain)   [ read/write | ^ ]  for  [ 1 day | ^ ]  to  "cookies, microphone, and camera"
 [x]  example.co.uk (trusted)         [ read/write | ^ ]  for  [ 1 day | ^ ]  to  "cookies, microphone, and camera"
 [x]  example.jp (trusted)               [ read/write | ^ ]  for  [ 1 day | ^ ]  to  "cookies, microphone, and camera"

Step 4: Promise returns with permission level, duration, and domain level accessor key


This isn't all encompassing, however it illustrates where the trust boundaries can potentially be improved and a possible mechanism for capturing user consent along with allowing users to fiddle with the request for consent by turning some things off. I would surely caution against isolating so many requests for access to a specific API as to be annoying and causing the users to "next next next" click like they do with set up wizards. I think the permissions go-between screens that SSO providers show when capturing consent represent a decent path to consent capture. This could also possibly be something that could be relayed to advertisers using TCF. Since the browser gets to choose the UI, the browser could also replace the word "trusted" with the word "advertiser/tracking" in the modal. 

So, in summary, restricting business shouldn't be the goal. Leveling the playing field shouldn't be the goal. Restricting the free-for-all all you can eat buffet of user identity and usage data and replacing it with trusted opt-in contract between user and publisher and allowing regional lawmakers to establish what constitutes fair use in transporting and exchanging data, which can't be regulated once it moves beyond the browser anyway, seems like a path forward.

**Comment by johannhof** [comment]
The current FPS proposal changes a lot relative to when this issue was filed, particularly regarding the new low size limit of associated subsets, so I'll close it as outdated. Feel free to re-open if needed.
